lib/index.js

```diff
@@ -33,7 +33,8 @@ module.exports = function rewireify(file, options) {
 
   function end() {
     post += "/* This code was injected by Rewireify */\n";
-    post += "if (Object.isExtensible(module.exports)) {\n";
+    post += "if ('functionobject'.match(typeof module.exports) && \n"
```

Comment:
Apologies, I interpreted it the wrong way around. I'll blame it on my brain is still warming up after holidays.

This is totally fine but readability should always trump terseness. To me it reads more naturally in the reverse order:

```
(typeof module.exports).match(/object|function/)
```

I'm not worried about any false positives such as matching an array, if that happened to be the case then the end user is may have larger problems to resolve.